### DISCUSSION TITLE
how did you know that it was a base64 text?

### DISCUSSION THREAD

**Discussion by Roshan-Ravindran** [vivian-dai/PicoCTF2021-Writeup#9] [Q&A] [answered]
because I couldn't figure it out by just looking at it

**Comment by vivian-dai** [comment, marked as answer]
In general, for bases, as long as you're exposed to them enough, you'll be able to recognize them pretty quickly. For base64, there's a few distinct features: 
* contains letters from A-Z (uppercase)
* contains letters from a-z (lowercase)
* contains digits from 0-9
* uses `+`, `/`, and `=` as padding
TLDR: learn that patterns that are distinct to each base
